target/riscv/cpu_helper.c

```diff
@@ -998,7 +1024,11 @@ hwaddr cpu_riscv_translate_address_tagmem(CPUArchState *env,
                                       &pmp_violation, &first_stage_error, prot,
                                       &pa, retpc);
     if (ret != TRANSLATE_SUCCESS) {
-        raise_mmu_exception(env, address, rw, pmp_violation, first_stage_error);
+        raise_mmu_exception(env, address, rw, pmp_violation,
+#ifndef TARGET_RISCV32
```

Comment:
Missing TARGET_CHERI.

Reply:
The entire function is already under the #ifdef.

Reply:
Ah I saw `cpu_riscv_translate_address` and didn't notice the `_tagmem`.